src/content/developers/docs/smart-contracts/deploying/index.md

```diff
@@ -60,6 +60,7 @@ Once deployed, your contract will have an Ethereum address like other [accounts]
 ## Related tutorials {#related-tutorials}
 
 - [Deploying your first smart contract](/developers/tutorials/deploying-your-first-smart-contract/) _– An introduction to deploying your first smart contract on an Ethereum test network._
+- [Hello World | smart contract tutorial](https://ethereum.org/ph/developers/tutorials/hello-world-smart-contract/) _– An easy-to-follow tutorial to create & deploy a basic smart contract on Ethereum._
```

Comment:
Hey @shelleyolivia, I think here you should remove the first part to not force the language :slightly_smiling_face: 
```suggestion
- [Hello World | smart contract tutorial](/developers/tutorials/hello-world-smart-contract/) _– An easy-to-follow tutorial to create & deploy a basic smart contract on Ethereum._
```

Reply:
Hey @vdusart, thanks for correcting the URL link and reviewing, noted for the future 😃